src/skirmishbunker/bunkerFloor.py

```diff
@@ -29,7 +29,7 @@ def make_interior_floor(self):
     columns = math_floor(int_width/(tile_size + tile_padding))
     rows = math_floor(int_length/(tile_size + tile_padding))
     tile_grid = grid.make_grid(part=floor_tile, dim = [tile_size + tile_padding, tile_size + tile_padding], columns = columns, rows = rows)
-    z_tile_translate = -1*(self.height/2-self.floor_tile_height/2-self.wall_width)
+    z_tile_translate = -1 * (self.height/2-self.floor_tile_height / 2 - self.floor_thickness)
```

Comment:
set a local variable default it to wall_thickness and swap it for floor_thickness when set